Example/Podfile.lock

```diff
@@ -255,7 +255,7 @@ SPEC CHECKSUMS:
   Artsy-UIButtons: 3c396f0fad352a7b0332100e0ffcb0ca577e0082
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   DoubleConversion: ebb6747c5b66026ad4f97b789c3ceac6f18e57a6
-  Emission: 8ffafeae9d93bdf019e60e8a21351f70d1d63ec7
+  Emission: f18165964b629d103bdba726db336903584eb2a3
```

Comment:
Err, so did my addition to the script not work?

Reply:
Looks like it to me alas

Reply:
Ok, I’ll have to think a bit longer about this then.

Reply:
Opened a ticket to address this: https://artsyproduct.atlassian.net/browse/LD-485

Reply:
Thanks!